examples/discovery-search-app/package.json

```diff
@@ -33,7 +35,8 @@
     "ibm-watson": "5.2.0-alpha-1",
     "prop-types": "^15.6.2",
     "react": "^16.4.1",
-    "react-dom": "^16.4.1"
+    "react-dom": "^16.4.1",
+    "ts-loader": "^6.2.1"
```

Comment:
I wonder if I can move these to dev dependencies at the root directory?

Reply:
this should be a dev dependency as it isn't required to run the example application